src/watertap_contrib/reflo/unit_models/zero_order/tests/test_chemical_softening.py

```diff
@@ -63,6 +63,7 @@ def chem_soft_frame(self):
             silica_removal=True,
             softening_procedure_type="excess_lime_soda",
         )
+        print(m.fs.soft.properties_in[0].is_property_constructed("flow_mol_phase_comp"))
```

Comment:
Can probably remove this now since not tested or make into an assert test

Reply:
oops good catch